packages/dds/sequence/README.md

```diff
@@ -170,3 +170,9 @@ Just like any other sequence, the Sparse Matrix is made of segments. The segment
     },
 ]
 ```
+
+# SharedObjectSequence and SharedNumberSequence
+
+Shared object sequence and share number sequence are very similar distrusted data structures. The only difference is their supported content. Shared number sequence only supports numbers as content, where shared object sequence supports any json serializable object. Both support inserting, removing, and annotating content. Each piece of content, number or object will occupy a single position in the sequence, and the length of the sequence represents to count of content items in the sequence.
```

Comment:
Is there ever a case in object sequence where the JSON object is "merged" with another JSON object? Or... is the merge logic higher level than the object itself? I guess from sequence's perspective all content within it is "content", and the merges far/near are at the content level.

Reply:
nope

Reply:
Are annotates covered elsewhere already? And they're merged like map values, correct? LWW?

Reply:
annotation is covered in this document, in the general sequence information.